app/uk/gov/hmrc/lisaapi/connectors/DesConnector.scala

```diff
@@ -56,7 +54,9 @@ class DesConnector @Inject()(
                     (implicit hc: HeaderCarrier): Future[DesResponse] = {
     val uri: String = s"$lisaServiceUrl/$lisaManager/investors"
     logger.debug("Posting Create Investor request to des: " + uri)
-    val result = wsHttp.POST[CreateLisaInvestorRequest, HttpResponse](uri, request)(implicitly, httpReads, updateHeaderCarrier(hc), implicitly)
```

Comment:
I know it's not something you changed but if we make the httpReads on line 42 implicit we can get rid of all the explicit passing of implicit parameters in this file? Fine to leave for now though if you don't wanna change it as it's probably out of scope here

Reply:
that's done